deploy-reports.ps1

```diff
@@ -49,7 +47,4 @@ try
 finally
 {
     $session.Dispose()
-}
-    
-
-ssh $UserName@$HostIP "powershell .\$ProjectLocation\create_tasks.ps1 -ProjectLocation $ProjectLocation -OutputDirectory $OutputDirectory"
```

Comment:
Is the idea then to run `deploy-report-schedules.ps1` separately? Or should this line be replace with an invocation to `deploy-report-schedules.ps1`? I think either is fine, just wanted to know what your plan was.

Reply:
I realized that we will have to update reports without updating their schedules so I split them.

Reply:
Ah yeah that makes sense. Good call!